tests/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: "2.2"
+name: pocam-test
+
+networks:
+  default:
+
+volumes:
+  site-public:
+  site-private:
+
+services:
+  mysql:
+    image: public.ecr.aws/unocha/mysql:10.6
+    hostname: pocam-test-mysql
+    container_name: pocam-test-mysql
+    environment:
+      - MYSQL_DB=pocam
+      - MYSQL_USER=pocam
+      - MYSQL_PASS=pocam
+    networks:
+      - default
+
+  drupal:
+    # Default to `odsg-site` for compatibility with the github tests.
+    # @see Makefile.
+    image: public.ecr.aws/unocha/${IMAGE_NAME:-pocam-site}:${IMAGE_TAG:-local}
+    hostname: pocam-test-site
+    container_name: pocam-test-site
+    depends_on:
+      - mysql
+    volumes:
+      - "./settings:/srv/www/shared/settings:ro"
+      # Mount volumes for the private and public files.
+      - "site-public:/srv/www/html/sites/default/files:rw"
+      - "site-private:/srv/www/html/sites/default/private:rw"
+      # Mount the folder for composer scripts.
+      - "../scripts:/srv/www/scripts:ro"
+      # Mount the folders needed for the tests.
+      - "./json:/srv/www/html/sites/default/files/json:rw"
+      - "./yaml:/srv/www/html/sites/default/files/content:rw"
```

Comment:
I don't think we have anything in the json and yaml folders, so they can be removed.